src/language-usability.rst

```diff
@@ -19,5 +19,133 @@ API Usability
 Program Comprehension
 =====================
 
+Software Visualization
+----------------------
+
+.. container:: bib-item
+
+  .. bibliography:: language-usability.bib
+    :filter: key == "price1993principled"
+
+The paper define **software visualization** as the use of the crafts of typography, graphic design, animation, and cinematography with modern human-computer interaction technology to facilitate both the human understanding and effective use of computer software. **Program visualization** is different from **software visualization** in that the latter includes the visualization of a high-level description of algorithm or a piece of software which is in contrast to **program visualization** where actual implemented code is visualized. This paper also gives a detailed "road map" of the work accomplished so far in **software visualization**, by identifying six broad categories of characteristics and by filling in the observed characteristics in each category.
```

Comment:
* "The paper define" -> "This paper defines"
* The sentence on program visualization can be moved to the section on that topic.
* What are the six broad categories of characteristics?